Packs/GoogleVault/Integrations/GoogleVault/GoogleVault.py

```diff
@@ -248,7 +248,7 @@ def create_mail_export_query(export_name, emails, time_frame, start_time, end_ti
     elif start_time:
         if not end_time:
             end_time = datetime.utcnow().isoformat() + 'Z'  # End time will be now, if no end time was given
-    if isinstance(emails, (str, unicode)):
+    if isinstance(emails, str):
```

Comment:
same for all
```suggestion
    if isinstance(emails, (str, bytes)):
```

Reply:
Because we check in the next line:
```python
if ',' in emails:
    emails = emails.split(',')
```
it can not be 'bytes'